extras/log_file_client/lib/utils/http_client.dart

```diff
@@ -231,52 +239,64 @@ class HttpClientTest extends HttpClient {
   // Fetches data from a hard-coded string (short) or the local Dart file (1440 sample lines) for testing purposes
   HttpClientTest();
 
-  late String testHTML =
-      '<html><body><ul><li><a href="/logs/ProjectA-tank-24.log">/logs/ProjectA-tank-24.log</a></li><li><a href="/logs/ProjectA-tank-70.log">/logs/ProjectA-tank-70.log</a></li><li><a href="/logs/ProjectB-tank-58.log">/logs/ProjectB-tank-58.log</a></li><li><ahref="/logs/index.html">/logs/index.html</a></li></ul></body></html>';
+  late String testHTML = '''
```

Comment:
Not needed for this PR, but eventually it would be nice to separate the "test" classes and code from the production code so that these strings don't form a part of the executable.